### PR TITLE
[3.0] fix issue #8695, exception log optimization

### PR DESCRIPTION
**Pull request by wangchengming666**
issue no #8695

### CODE REVIEW

File: dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/exchange/support/DefaultFuture.java [hunk 1]
```diff
@@ -251,11 +251,11 @@ private String getTimeoutMessage(boolean scan) {
                 + (sent > 0 ? " client elapsed: " + (sent - start)
                 + " ms, server elapsed: " + (nowTimestamp - sent)
                 : " elapsed: " + (nowTimestamp - start)) + " ms, timeout: "
-                + timeout + " ms, request: " + (logger.isDebugEnabled() ? request : getRequestWithoutData()) + ", channel: " + channel.getLocalAddress()
+                + timeout + " ms, request: " + (logger.isDebugEnabled() ? request : getRequestWithoutData(request)) + ", channel: " + channel.getLocalAddress()
                 + " -> " + channel.getRemoteAddress();
     }
 
-    private Request getRequestWithoutData() {
+    private static Request getRequestWithoutData(Request request) {
```
Comment (plusmancn) [inline dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/exchange/support/DefaultFuture.java:258]: Don't use static.

Static results in many problems during the multi-instance refactor.

Comment (changfubai) [reply dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/exchange/support/DefaultFuture.java:258]: would you mind explaining it? I think this useage is not perfect, but I don't understand the meaning of `Static results in many problems during the multi-instance refactor.`
please help me, thanks.

Comment (wangchengming666) [reply dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/exchange/support/DefaultFuture.java:258]: I think private static method it's ok.

**Comment by AlbumenJ** [comment]
fixed in #8775

**Pull request closed without merging**